### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by plarocque4**
## Describe your changes

Update the rems-admin README

## Issue ticket number and Jira link

[REMS-701](https://jira.mitre.org/browse/REMS-701)

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] Ensure the target / base branch for any feature PR is set to `dev` not main (the only exception to this is releases from `dev` and hotfix branches)

## Checklist for conducting a review
- [ ] Review the code changes and make sure they all make sense and are necessary. 
- [ ] Pull the PR branch locally and test by running through workflow and making sure everything works as it is supposed to. 

## Workflow 

Owner of the Pull Request will be responsible for merge after all requirements are met, including approval from at least one reviewer. Additional changes made after a review will dismiss any approvals and require re-review of the additional updates. Auto merging can be enabled below if additional changes are likely not to be needed. The bot will auto assign reviewers to your Pull Request for you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,42 +1,31 @@
...
+
+### `git submodule update --init`
+
+Next, install the required dependencies by running the following:
+
+### `npm install`
 
 ## Running the Mongo DB instance
 
+The REMS Administrator relies on MongoDB for it's backing database.
```
Comment (KeeyanGhoreshi) [inline README.md:27]: Do the docker commands here actually work?  My experience has been that only the ones in rems-setup work correctly.

File: README.md [hunk 2]
```diff
@@ -54,42 +43,71 @@ Following are a list of modifiable paths:
...
+Launches the test runner in the interactive watch mode.\
+See the section about [running tests](https://create-react-app.dev/docs/running-tests/) for more information.
 
-```
-npm install
-```
+## Usage
 
-#### Run Tests
+The REMS Admin interacts with the [Request Generator](https://github.com/mcode/request-generator), [REMS SMART on FHIR app](https://github.com/mcode/rems-smart-on-fhir), and an [EHR](https://github.com/mcode/test-ehr). These apps are provided as part of the REMS ecosystem, but any individual part may be swapped out for something custom. The REMS Admin responsds to CDS Hooks request as well as FHIR operations for the questionnaire package ($questionnaire-package) and REMS ETASU check ($rems-etasu).
```
Comment (KeeyanGhoreshi) [inline README.md:64]:
```suggestion
The REMS Admin interacts with the [Request Generator](https://github.com/mcode/request-generator), [REMS SMART on FHIR app](https://github.com/mcode/rems-smart-on-fhir), and an [EHR](https://github.com/mcode/test-ehr). These apps are provided as part of the REMS ecosystem, but any individual part may be swapped out for something custom. The REMS Admin responds to CDS Hooks requests as well as FHIR operations for the questionnaire package ($questionnaire-package) and REMS ETASU check ($rems-etasu).
```

File: README.md [hunk 3]
```diff
@@ -54,42 +43,71 @@ Following are a list of modifiable paths:
...
+| AUTH_SERVER_URI | `http://localhost:8090`                    | The base url of the auth server, currently set to the base url of this app.                           |
+| HTTPS_CERT_PATH | `server.cert`                              | Path to a certificate for encryption, allowing HTTPS. Unnecessary if using HTTP.                      |
+| HTTPS_KEY_PATH  | `server.key`                               | Path to a key for encryption, allowing HTTPS. Unnecessary if using HTTP.                              |
+| LOGGING_LEVEL   | `debug`                                    | Amount to output in the log, can be changed to verbose, info, warn, or error.                         |
+| MONGO_DB_NAME   | `remsadmin`                                | Name of the database table being used. Should be changed if not using the Mongo instructions below.   |                 
+| MONGO_URL       | `mongodb://rems-user:pass@127.0.0.1:27017` | URL for the connection to the database, should be changed if not using the Mongo instructions below.  |
```
Comment (KeeyanGhoreshi) [inline README.md:107]: The mongo instructions are above, in this case

File: README.md (hunk 1 above)
Comment (plarocque4) [reply README.md:27]: good question, I'll update this to show how to launch mongo without docker

File: README.md (hunk 3 above)
Comment (plarocque4) [reply README.md:93]: updated

File: README.md [hunk 4]
```diff
@@ -1,95 +1,99 @@
...
+| AUTH_SERVER_URI | `http://localhost:8090`                    | The base url of the auth server, currently set to the base url of this app.                           |
+| HTTPS_CERT_PATH | `server.cert`                              | Path to a certificate for encryption, allowing HTTPS. Unnecessary if using HTTP.                      |
+| HTTPS_KEY_PATH  | `server.key`                               | Path to a key for encryption, allowing HTTPS. Unnecessary if using HTTP.                              |
+| LOGGING_LEVEL   | `debug`                                    | Amount to output in the log, can be changed to verbose, info, warn, or error.                         |
+| MONGO_DB_NAME   | `remsadmin`                                | Name of the database table being used. Should be changed if not using the Mongo instructions above.   |                 
+| MONGO_URL       | `mongodb://rems-user:pass@127.0.0.1:27017` | URL for the connection to the database, should be changed if not using the Mongo instructions below.  |
```
Comment (KeeyanGhoreshi) [inline README.md:93]:
```suggestion
| MONGO_URL       | `mongodb://rems-user:pass@127.0.0.1:27017` | URL for the connection to the database, should be changed if not using the Mongo instructions above.  |
```

**Review by KeeyanGhoreshi — APPROVED** [review]

**Pull request merged**